airgun/entities/rhai/manage.py

```diff
@@ -0,0 +1,39 @@
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.rhai import Manage
+
+
+class ManageEntity(BaseEntity):
+
+    def _toggle_service(self, state):
+        view = self.navigate_to(self, "Details")
+        if view.enable_service.fill(state):
+            view.save.click()
+
+    def enable_service(self):
+        self._toggle_service(True)
+
+    def disable_service(self):
+        self._toggle_service(False)
+
+    @property
+    def status(self):
```

Comment:
There are no "hundred" methods, only these two properties.

Reply:
right, but the idea behind any approach, to keep rules even if that is not optimal per specific situation, because that will ruin it in future. I completely understand your point, but I just want to be consistent